WorkshopToolCommon/DiscordWebhook.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Phoenix.WorkshopTool
+{
+    class DiscordWebhook
+    {
+        WorkshopType m_type;
+        string m_title;
+        string m_changelog;
+
+        //it is a small payload
+        static readonly string _payload = @"{{""embeds"":[ {{""author"": {{""name"": ""{0}""}}, ""title"": ""{1}"", ""description"": ""{2}""}} ]}}";
```

Comment:
I missed this the first time, strings are immutable, so this should be const instead.
Not an issue though.